docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.horizontal-card.scss

```diff
@@ -0,0 +1,129 @@
+.horizontal-card {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 2.75rem;
```

Comment:
We should remove the margin here and rely on layout and utility classes for spacing between cards. This will let the card be a bit more flexible in other scenarios.